test/security/PauseManager.t.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.16;
+
+import "forge-std/Test.sol";
+import "../../contracts/security/PauseManager.sol";
+import "../../contracts/mocks/tokens/MockTestnetToken.sol";
+
+contract PauseManagerTest is Test {
```

Comment:
Add test 
- testShouldRevertAddGuardianWhenNotOwner
- testShouldRevertRemoveGuardianWhenNotOwner

Reply:
I cannot add these tests here because the library has no owner, and these tests are already in Stanley's tests.